packit/schema.py

```diff
@@ -108,7 +108,7 @@ def _deserialize(
         **kwargs,
     ) -> Dict:
         if not isinstance(value, dict):
-            raise ValidationError("Invalid data provided. dict required")
+            raise ValidationError("Invalid data provided (type dict required).")
```

Comment:
```suggestion
            raise ValidationError(f"'dict' required, got {type(value)!r}.")
```

would be more useful.